spec/models/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'User' do
+  subject(:user) do
+    User.new(
+      first_name: first_name, last_name: last_name, email: email, password: password,
+      password_confirmation: password_confirmation
+    )
+  end
+
+  let(:first_name)            { Faker::Name.first_name }
+  let(:last_name)             { Faker::Name.last_name }
+  let(:email)                 { Faker::Internet.email }
+  let(:password)              { 'password' }
+  let(:password_confirmation) { 'password' }
+
+  context 'With an empty user' do
+    it '#attr_getters' do
+      %w[first_name last_name].each do |un_atr|
+        expect(user.attributes.keys).to include(un_atr)
+      end
+    end
+
+    it do
+      is_expected.to be_valid
+    end
+  end
+
+  describe '#generate_verification_code' do
+    context 'When the user is created' do
+      it 'generates a random verification code' do
```

Comment:
I don't remember anything about a verification code. The code snippet listed in the card was only an example, you don't need to copy-paste as it is.